src/components/NanoContract/NanoContractIcon.svg.component.js

```diff
@@ -0,0 +1,21 @@
+import * as React from "react"
+import Svg, { SvgProps, Path } from "react-native-svg"
+/**
```

Comment:
```suggestion
import * as React from 'react';
import Svg, { SvgProps, Path } from 'react-native-svg';

/**
```

Reply:
Actually, the whole file is using double quotes

Is your editor's LSP loading the `.eslintrc`? And is lint passing here?

Reply:
It was fixed